frontend/src/app/shared/custom/scoring/scoring.component.ts

```diff
@@ -33,16 +33,13 @@ export class ScoringComponent implements OnInit, AfterViewInit, OnChanges {
   targetPercent: number = 0;
   labelPercentage: Observable<number>;
   stretched: boolean = false;
-  protected readonly isLastCheckInNegative = isLastCheckInNegative;
```

Comment:
why is this removed? don't we need this for the exclamation mark to be set for negative checkins?

Reply:
looks like this got renamed to calculateCurrentPercentage with a check for <1